homeassistant/components/samsungtv/config_flow.py

```diff
@@ -81,22 +93,23 @@ def _get_entry(self):
             },
         )
 
-    def _try_connect(self):
-        """Try to connect and check auth."""
-        for cfg in SUPPORTED_METHODS:
+    def _try_connect_samsungctl(self, port):
```

Comment:
When we decide to use samsungctl only for legacy is there a reason to choose a differenr port? I was always wondering if there ever was one.

Reply:
Sorry I do not understand your question

Reply:
Legacy is working on port 55000 and I don't see any reason to configure a different one. Additionally samsungctl doesn't even need the port parameter. Yes I know, it was always there but we should set it fixed.